src/scripts/main.js

```diff
@@ -358,3 +358,33 @@ const people = [
 console.log(people); // you can remove it
```

Comment:
You have a console.log statement here. While this is not a problem per se, it's good practice to remove console.log statements from your code before committing, as they are generally used for debugging and are not necessary in production code.